src/writeData/components/WriteDataDetailsContext.tsx

```diff
@@ -65,7 +71,7 @@ const WriteDataDetailsProvider: FC = ({children}) => {
   }, [variables, org?.name])
 
   useEffect(() => {
-    if (bucket?.name === variables.bucket) {
+    if (!bucket || bucket?.name === variables.bucket) {
```

Comment:
These aren't equivalent?

Reply:
they should be but it was breaking for me if the bucket was undefined.

Reply:
![Kapture 2021-05-19 at 13 19 24](https://user-images.githubusercontent.com/1637395/118856471-fafd8c80-b8a4-11eb-8e7d-d96481be55a8.gif)